sales/amazon_connector/new_marketplace.rst

```diff
@@ -0,0 +1,40 @@
+======================================================
+Add an unsupported marketplace to the Amazon Connector
+======================================================
+
+Some newer Amazon Marketplaces, such as Amazon Brazil or Amazon Netherlands, are
```

Comment:
I'm not sure about "newer". The Brazil marketplace is not know I think, just limited to sellers based in Brazil.